src/main/java/com/zendesk/maxwell/schema/ddl/SchemaChange.java

```diff
@@ -58,7 +58,7 @@ private static boolean matchesBlacklist(String sql) {
 		// now strip out comments
 		sql = CSTYLE_COMMENTS.matcher(sql).replaceAll("");
 		sql = sql.replaceAll("\\-\\-.*", "");
-		sql = sql.replaceAll("^\\s*#.*", "");
+		sql = Pattern.compile("^\\s*#.*", Pattern.MULTILINE).matcher(sql).replaceAll("");
```

Comment:
uh this doesn't seem right to me now that i look closer at it -- this regex will strip out all the sql if anything starts with `#`.

Reply:
sql to this point should be ddl, sql starting with #, is there any other use? The processing method before my test really cannot handle multiple lines of sql with "newline character" in one line of sql and starting with #. We should handle this problem. I hope to get your advice

Reply:
i take it back, your PR is fine.  it looks dangerous but it's not.

Reply:
I hope I can get some help to solve this problem